pkg/config/config.go

```diff
@@ -1558,6 +1567,9 @@ func setupFipsEndpoints(config Config) error {
 	// Logs
 	config.Set("logs_config.use_http", true)
 	config.Set("logs_config.logs_no_ssl", true)
+	if config.GetBool("fips.https") {
+		config.Set("logs_config.logs_no_ssl", false)
+	}
```

Comment:
```suggestion
		config.Set("logs_config.logs_no_ssl", !config.GetBool("fips.https"))
```

Reply:
This is the exact change I pushed earlier today prior to checking your review.